### PR TITLE
fix: [PL-48009]: bump uicore version

### PR DESCRIPTION
**Pull request by namanharness**
Bump version for this PR: https://github.com/harness/uicore/pull/1250

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

File: packages/uicore/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harness/uicore",
-  "version": "3.159.2",
+  "version": "3.156.0",
```
Comment (MondalSayantan) [inline packages/uicore/package.json:3]: Please correct it to`3.159.3`

**Review by MondalSayantan — CHANGES_REQUESTED** [review]

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2369

File: packages/uicore/package.json (hunk 1 above)
Comment (namanharness) [reply packages/uicore/package.json]: Thanks for noticing this!

**Review by MondalSayantan — APPROVED** [review]

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2370

**Review by chetanvekariya-harness — APPROVED** [review]

**Pull request merged**